content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1040,6 +1040,12 @@ N/A
 @id
 ```
 
+As of 2.29.0, if the `namedConstraints` preview flag is enabled the signature changes:
+
+```ts no-lines
```

Comment:
```suggestion
```prisma no-lines
```

Reply:
I notice that some of the other examples here are indeed `ts`, but some are `prisma`. Is there a significance to this @molebox? Looks very `prisma` to me :D

Reply:
If its the PSL then we should use `prisma` otherwise, we should use the code that is being used.

Reply:
Those are all PSL.

Reply:
Then they should all be prisma 😅